content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -51,6 +51,21 @@ may [fail](https://github.com/kubernetes/kubeadm/issues/31).
 If you have more than one network adapter, and your Kubernetes components are not reachable on the default
 route, we recommend you add IP route(s) so Kubernetes cluster addresses go via the appropriate adapter.
 
+## Ensure iptables does not use the nf_tables backend
+
+On Debian 10 (Buster) and other recent distributions, the `iptables` binary uses the newer `nf_tables`
+backend by default. This backend is not compatible with current Kubernetes releases: it causes duplicated
+firewall rules and breaks `kube-proxy`.
+
+To avoid these problems, switch `iptables` and related binaries back to the 'legacy' backend:
+
+```
+update-alternatives --set iptables /usr/sbin/iptables-legacy
+update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
+update-alternatives --set arptables /usr/sbin/arptables-legacy
+update-alternatives --set ebtables /usr/sbin/ebtables-legacy
```

Comment:
FTR, on Fedora, that's:

    alternatives --set iptables /usr/sbin/iptables-legacy

(Only one command is needed, not 4.)
I believe that's also correct on SUSE (um... @mrostecki ?)

Reply:
How about putting the instructions in tabs per OS family?

Reply:
@danwinship to which Fedora and (open)SUSE versions does this apply?

Reply:
I've added instructions for RHEL/CentOS/Fedora in tabs :slightly_smiling_face:

Reply:
I don't believe RHEL 8 supports `iptables-legacy` at all. See https://access.redhat.com/solutions/4377321 which states that legacy support was dropped.